templates/docker-compose.template

```diff
@@ -59,20 +56,21 @@ services:
     environment:
       - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
       - FIRMWARE_SHORT_HASH={{ENV.FIRMWARE_SHORT_HASH}}
+    ports:
+      - "1681:1681"
```

Comment:
we should leave as expose not ports...as we dont want to expose externally

Reply:
AFAIK expose exposes to other containers but ports exposes them external to the device - which we dont want

Reply:
I've checked. expose is now used decoratively. The functionality is the same as forwarding. That's why I've used it under ports.

Reply:
Ok done.